src/testpress/video_watermark/includes/form.html

```diff
@@ -1,4 +1,3 @@
-<div class="flex w-full flex-wrap">
```

Comment:
Remove everything in this form and start fresh. Copy the form UI from tailwind and use it without changing any classes.